pkgs/development/python-modules/taxi/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchPypi, appdirs, requests, click, setuptools }:
+
+buildPythonPackage rec {
+  pname = "taxi";
+  version = "6.1.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "taxi";
+    sha256 = "b2562ed58bd6eae7896f4f8e48dbee9845cd2d452b26dd15c26f839b4864cb02";
+  };
+
+  doCheck = false;
```

Comment:
Can you add a comment explaining why the tests have to be disabled? 
It would make things easier for future maintainers